### PR TITLE
metazoan species Cimex lectularius static content updated for 105

### PR DESCRIPTION
**Pull request by vsitnik**
updated content for updated Cimex lectularius in e105

### CODE REVIEW

**Comment by ens-ds23** [comment]
Thanks @vsitnik . This is fine.

**Review by ens-ds23 — APPROVED** [review]

**Comment by vsitnik** [comment]
I need  to add a couple of fixes. Sorry @ens-ds23

**Comment by vsitnik** [comment]
@ens-ds23  done. Sorry

**Comment by ens-ds23** [comment]
All fine. Thanks @vsitnik . Will merge in now.

File: metazoa/species/Cimex/Cimex_lectularius/Cimex_lectularius_about.md [hunk 1]
```diff
@@ -12,8 +12,10 @@ to be a vector although recent studies have shown that it can transmit
```
Comment (ens-ds23) [inline metazoa/species/Cimex/Cimex_lectularius/Cimex_lectularius_about.md:12]: confirmed this diff now on staging.

File: metazoa/species/Cimex/Cimex_lectularius/Cimex_lectularius_annotation.md [hunk 2]
```diff
@@ -1,4 +1,4 @@
-*ClecH1.3* gene set
```
Comment (ens-ds23) [inline metazoa/species/Cimex/Cimex_lectularius/Cimex_lectularius_annotation.md:1]: Confirmed this diff now on staing.

File: metazoa/species/Cimex/Cimex_lectularius/Cimex_lectularius_assembly.md [hunk 3]
```diff
@@ -1,5 +1,4 @@
-*ClecH1* assembly
```
Comment (ens-ds23) [inline metazoa/species/Cimex/Cimex_lectularius/Cimex_lectularius_assembly.md:1]: Confirmed this diff now on staging.

File: metazoa/species/Cimex/Cimex_lectularius/Cimex_lectularius_references.md [hunk 4]
```diff
@@ -1,8 +1,6 @@
 ### References
```
Comment (ens-ds23) [inline metazoa/species/Cimex/Cimex_lectularius/Cimex_lectularius_references.md:1]: Confirmed this diff now on staging.

File: metazoa/whatsnew.md [hunk 5]
```diff
@@ -1,48 +1,6 @@
 - Updated data
```
Comment (ens-ds23) [inline metazoa/whatsnew.md:1]: Confirmed this diff now on staging.

**Review by ens-ds23 — COMMENTED** [review]
All live on staging.

**Pull request merged**